### PR TITLE
Rework README

### PR DESCRIPTION
**Pull request by crswty**
TLDR; Updated README, see it [here](https://github.com/crswty/postfacto/tree/feat-new-readme)

The readme is currently the main entrypoint to the project and I think that currently it's not too clear to potential users what it is or how they would go about using it.

To that end, this PR reworks the readme to follow what I think would be the flow for someone new landing on the page, i.e:
1. What is this thing?
1. Why would I use it?
1. How do I use it?
1. How do I contribute

I've added a gif that runs from the homepage through to running a retro as well as updated the feature list.

I've also made it easier to see the deployment instructions for each platform without having to download the packages as I think that's generally expected.

Thanks for contributing to postfacto. To speed up the process of reviewing your pull request please provide us with:

* [x] I have reviewed the [contributing guide](https://github.com/pivotal/postfacto/blob/master/CONTRIBUTING.md)

* [x] I have made this pull request to the `master` branch

* [ ] I have run all the tests using `./test.sh`.

* [ ] I have added the [copyright headers](https://github.com/pivotal/postfacto/blob/master/license-header.txt) to each new file added

* [ ] I have given myself credit in the [humans.txt](https://github.com/pivotal/postfacto/blob/master/humans.txt) file (assuming I want to)

### CODE REVIEW

**Comment by crswty** [comment]
@liamdarmody Think you're probably the best person to look at this, what do you think?

File: README.md [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
-Download the [latest release](https://github.com/pivotal/postfacto/releases) and follow the included instructions on deploying your own instance of Postfacto. 
-
-_Note: Postfacto **does not** support being deployed from source, it must be deployed from a packaged version/archived release._
 
 ## Features
 
-### Run retros remotely
+### Run Retros Remotely
+Postfacto lets you run an agile retrospective even when your team is distributed across the world. Retros live update across all your devices
+so each participant can follow along on their device
```
Comment (liamdarmody) [inline README.md]:
```suggestion
so each participant can follow along on their device.
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,50 @@
...
 
-Postfacto lets you run an agile retrospective even when your team is distributed across the world.
+### Record Action Items
+Retros are designed to help the team improve and that's hard to do without keeping track of the actions the team needs to carry 
+out and the also the one's you've achieved. Postfacto tracks the items you enter week to week to help you keep on top of them.
 
-### Record action items
+### Mobile Friendly
+Participants can add and vote on items from their mobile devices, so its easy to run a retro without everyone in the group having a laptop nearby. This works really well for retros where some people 
+are in the room and some are remote
```
Comment (liamdarmody) [inline README.md]:
```suggestion
are in the room and some are remote.
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,50 @@
...
 
-Retros are designed to help the team improve and that's hard to do without keeping track of the actions the team needs to carry out and the also the one's you've achieved. Postfacto tracks the items you enter week to week to help you keep on top of them.
+### Easy Onboarding
+Easily [set up](https://github.com/pivotal/postfacto/tree/master/deployment#allowing-users-to-create-retros) Postfacto to work with Google OAuth so users can sign up with their google accounts
```
Comment (liamdarmody) [inline README.md]:
```suggestion
Easily [set up](https://github.com/pivotal/postfacto/tree/master/deployment#allowing-users-to-create-retros) Postfacto to work with Google OAuth so users can sign up with their google accounts.
```
